raspicheck.py

```diff
@@ -0,0 +1,69 @@
+import discord
+from discord.ext import commands
+from __main__ import set_cog, send_cmd_help, settings
+import asyncio
+import psutil

```

Comment:
I believe this needs to be pip-installed. There's a new way to do so:
![image](https://cloud.githubusercontent.com/assets/17536161/23283737/64a0ed0c-fa73-11e6-8b80-a52d8f4e8a21.png)